IBAnimatable/GradientDesignable.swift

```diff
@@ -78,14 +46,48 @@ fileprivate extension GradientDesignable where Self: UIView {
   }
 
   func configurePredefinedGradient() -> GradientColor? {
-
     guard let gradientType = predefinedGradient else {
       return nil
     }
     return gradientType.colors
   }
 }
 
+public extension GradientDesignable where Self: UINavigationBar {
+  public func configureGradient() {
+    let predefinedGradient = configurePredefinedGradient()
+    if let startColor = startColor, let endColor = endColor {
+      configureGradient(startColor: startColor, endColor: endColor)
+    } else if let startColor = predefinedGradient?.start, let endColor = predefinedGradient?.end {
+      configureGradient(startColor: startColor, endColor: endColor)
+    }
+  }
+}
+
+fileprivate extension GradientDesignable where Self: UINavigationBar {
```

Comment:
@tbaranes I think started to use the private extension to group private methods like `fileprivate extension GradientDesignable where Self: UIView`. The original idea may be from https://github.com/raywenderlich/swift-style-guide#code-organization for using the `extension`s to organise the code.